cmd/edgectl/cli.go

```diff
@@ -235,7 +235,7 @@ func (d *Daemon) getRootCommand(p *supervisor.Process, out *Emitter, data *Clien
 	})
 	intercept := InterceptInfo{}
 	interceptAddCmd := &cobra.Command{
-		Use:   "add DEPLOYMENT [--namespace NAMESPACE] [-p PREFIX] -t [HOST:]PORT -m HEADER=REGEX ...",
+		Use:   "add DEPLOYMENT -t [HOST:]PORT -m HEADER=REGEX ...",
```

Comment:
Why are namespace and prefix not worth mentioning here anymore?

Reply:
They are optional. Since `edgectl intercept add` is a bit complicated, I chose to show what is _required_ in the usage section, leaving the optional stuff for the flags section. Those two options snuck in when we scrambled to fix namespace stuff after the Agent-_is_-AES rewrite, and I missed it in my review.